modules/javafx.web/src/main/native/Source/WebCore/page/FrameViewLayoutContext.cpp

```diff
@@ -386,7 +386,13 @@ void FrameViewLayoutContext::scheduleLayout()
         LOG(Layout, "FrameView %p layout timer scheduled at %.3fs", this, frame().document()->timeSinceDocumentCreation().value());
 #endif
 
+#if PLATFORM(JAVA)
+    // scheduleLayout will be called from prePaint in next updateContent cycle.
+    const Seconds layoutScheduleThreshold = 250_ms;
+    m_layoutTimer.startOneShot(layoutScheduleThreshold);
+#else
```

Comment:
I presume there was some change in the imported WebKit commits that made this changes necessary?

Reply:
That is a good question. I believe that was actually introduced because of JDK-8260165 (and removed in the next update by commit ed0baf5f23aed0d8aaa72645c8e03fde56d0f0cc) 

The problem with this 250ms delay is that in case the WebView is not visible, it's contents won't get updated until after 250ms. That is a major pain if the WebView is used for printing only (hence not for rendering).

Reply:
I think it was missed during webkit update, and caused [JDK-8269067](https://bugs.openjdk.java.net/browse/JDK-8269067) : CSSFilterTest fails intermittently in Windows. 
Taking this change back seems like good idea.

Reply:
The original addition of this #ifdef was due to JDK-8260165. That commit introduced the problems I mentioned above (WebView took 250ms to update if not visible), so I don't think it is a good solution. 
I suggest we discuss that in JDK-8269067 (but not include it in the PR with the cherry-pick of the changes, as it is not in the upstream webkit)?

Reply:
I have reverted this change. The CSSFilterTest.testCSSFilterRendering is already ignored.